examples/a50t_all_features/README.md

```diff
@@ -0,0 +1,43 @@
+This inserts all(3948533) features of the xc7a50t in a vector.
+
+To get resource usage, run with `make run`, which uses `/usr/bin/time` to measure running time and peak memory.
+
+Current output:
+```
+=== Tables ===
+--- 0 level (2149 entries)---
+...
+0 of the 2148 strings are stored in the pointer.
+--- 1 level (1246 entries)---
+...
+200 of the 1245 strings are stored in the pointer.
+--- 2 level (1448 entries)---
+...
+119 of the 1447 strings are stored in the pointer.
+--- 3 level (4 entries)---
+...
+0 of the 3 strings are stored in the pointer.
+	Command being timed: "./main"
+	User time (seconds): 1.62
+	System time (seconds): 0.08
+	Percent of CPU this job got: 81%
+	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:02.09
```

Comment:
So with hashing, this drops to 2 minutes, 9 seconds - compared to 12 m previously?

Reply:
looks like GNU time has a weird time notation... the program runs in 2 seconds. this is expected because we are doing ~1000x less strcmp's than before.